cypress/integration/jsEmployeesTableDOM.spec.js

```diff
@@ -0,0 +1,146 @@
+'use strict';
+
+Cypress.Commands.add('fillTable',
+  (fullName, position, city, age, salary) => {
+    cy.get('[data-qa="name"]').type(fullName);
+    cy.get('[data-qa="position"]').type(position);
+    cy.get('[data-qa="city"]').select(city);
+    cy.get('[data-qa="age"]').type(age);
+    cy.get('[data-qa="salary"]').type(salary);
+    cy.get('button').contains('Save to table').click();
+  });
+
+Cypress.Commands.add('checkData',
+  (fullName, salary) => {
+    cy.get('tbody').contains(fullName).should('not.exist');
+    cy.get('tbody').contains(salary).should('not.exist');
+  });
+
+describe('Employees table', () => {
+  beforeEach('Open site', () => {
+    cy.visit('/');
+  });
+
+  it('should sort by name ASC', () => {
+    cy.get('th').contains('Name').click();
+
+    cy.get('tbody > :nth-child(1) > :nth-child(1)')
+      .contains('Airi Satou');
```

Comment:
test whole data is sorted, not just first row is correct.

Reply:
got it